sklearn/impute.py

```diff
@@ -141,6 +141,19 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
     statistics_ : array of shape (n_features,)
         The imputation fill value for each feature.
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from sklearn.impute import SimpleImputer
+    >>> X = [[np.nan, 2, 3], [4, np.nan, 6], [10, np.nan, 9]]
+    >>> imp_mean = SimpleImputer(missing_values=np.nan, strategy='mean')
+    >>> imp_mean.fit([[7, 2, 3], [4, np.nan, 6], [10, 5, 9]])
+    >>> print(imp_mean.transform(X))
+        # doctest: +NORMALIZE_WHITESPACE
```

Comment:
The tests are failing because you need to write all the outputs.
```python
>>> imp_mean = SimpleImputer(missing_values=np.nan, strategy='mean')
SimpleImputer(copy=True, fill_value=None, missing_values=nan, strategy='mean',
              verbose=0)
```